Installation_ubuntu16.md

```diff
@@ -0,0 +1,64 @@
+# 1. install nvm/node
+```
+$ cd ~/
+$ git clone https://github.com/creationix/nvm.git .nvm
+$ cd ~/.nvm 
+$ git checkout v0.33.8
```

Comment:
Magic number

Reply:
v0.33.8？

Reply:
nvm is not a required component. Following the official Nodejs installation links README given is enough.